index.js

```diff
@@ -50,7 +50,8 @@ var cssGlobbingPlugin = function(options) {
   return map(function(code, filename) {
 
     var content = code.toString();
-    var importRegExp = /^\s*@import\s+((?:url\()?["']?)?([^"'\)]+)(['"]?(?:\))?)?;\s*$/gm;
+    var semicolon = filename.indexOf('.sass') !== -1 ? '' : ';';
```

Comment:
I would prefer to use `path.extname` here instead, for clarity and since we're `require`:ing `path` in this file anyways.